.env

```
@@ -1,4 +1,5 @@
 NODE_ENV='production'
+IDENTIFIER_2U='2U'
```

Let's rename this as well, and remove the "2U" default setting which does not make sense for all use cases.

```suggestion
COURSE_TYPE_FILTER=''
```